tests/vpn/openvpn/task.yaml

```
@@ -42,4 +44,10 @@ execute: |
   fi
 
   network-manager.nmcli c down default
+
+  # Check that tun0/tun1/tun2 temporary connections do not have a static config file
+  if grep 'name: "tun' /etc/netplan/*; then
```

Suggestion (non-blocking): Be a bit more flexible with checking for quoted or unquoted "tun*" names and check inside `*.yaml` files only.
```suggestion
  if egrep 'name: "?tun' /etc/netplan/*.yaml; then
```